.github/workflows/error-codes-updater.yaml

```
@@ -0,0 +1,65 @@
+name: Consul Error Codes Utility Runner
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - '**.go'
+
+jobs:
+  Update-error-codes:
+    name: Error codes utility
+    if: github.repository == 'meshery/meshery-consul'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        # token here with write access to consul repo
+        with:
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          ref: 'master'
+
+      - name: Setup Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ secrets.GO_VERSION }}
+
+      - name: Run utility
+        run: |
+          go run github.com/meshery/meshery-consul/cmd/errorutil -d . update --skip-dirs meshery -i ./helpers -o ./helpers
```

```suggestion
          go get github.com/layer5io/meshkit/cmd/errorutil
          go run github.com/layer5io/meshkit/cmd/errorutil -d . update --skip-dirs meshery -i ./helpers -o ./helpers
```
@iamsdas Actually this command fetches the error codes utility from meshkit and runs it. Please don't change this : )
Also, I just realized that consul adapter still doesn't use the latest meshkit so we'll need to fetch the latest release on the action runner itself.

---

@DelusionalOptimist, I made the suggested changes, anything else?